src/components/navs/AppNav/AppNavAccountBtn.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <BalPopover no-pad>
+  <BalPopover
+    no-pad
+    :align="isMobile ? 'right' : undefined"
+    :detached="isMobile ? true : undefined"
```

Comment:
Since I only need these on mobile - passing `undefined` would basically mean these props won't be passed down.